### PR TITLE
Fix tracking

### PR DESCRIPTION
**Pull request by amarts**
Noticed that GA tracking was not working, looks like order of parameters matter in tracking.

### CODE REVIEW

**Comment by amarts** [comment]
Now, everything is working. Had made a mistake in GA tracking ID. With this, that gets fixed.

File: operator/main.py [hunk 1]
```diff
@@ -315,16 +314,17 @@ def deploy_storage_class():
     logging.info(logf("Deployed StorageClass", manifest=filename))
 
 def send_analytics_tracker(ga_id):
-    reqheader = { 'user-agent': 'Kadalu-App', 'accept': '*/*' }
+    reqheader = { 'user-agent': 'Mozilla/5.0 (Kadalu-App)', 'accept': '*/*' }
     url = "https://www.google-analytics.com/collect?v=1&t=pageview"
-    epoch = time.time()
     track_page = "http://kadalu.org/kadalu-operator"
```
Comment (aravindavk) [inline operator/main.py]: We can also add version here. (Note: `KADALU_VERSION` env to be added to server and csi yaml.j2 files.)

```
track_page = "http://kadalu.org/kadalu-%s-%s" % (name, os.environ.get("KADALU_VERSION", "latest"))
```

File: operator/main.py [hunk 2]
```diff
@@ -315,16 +314,17 @@ def deploy_storage_class():
     logging.info(logf("Deployed StorageClass", manifest=filename))
 
 def send_analytics_tracker(ga_id):
```
Comment (aravindavk) [inline operator/main.py]: If possible we can move this function to `$SRC/lib/kadalulib.py`

Comment (amarts) [reply operator/main.py]: Yes, thats a good idea. We would reduce the mistakes.

File: operator/main.py (hunk 1 above)
Comment (amarts) [reply operator/main.py]: Ack!

Comment (amarts) [reply operator/main.py]: Done

File: operator/main.py (hunk 2 above)
Comment (amarts) [reply operator/main.py]: Done

**Review by aravindavk — APPROVED** [review]

**Review by aravindavk — APPROVED** [review]

**Pull request merged**